equinix-openapi-metal/docs/Interconnection.md

```diff
@@ -11,7 +11,7 @@
 |**description** | **String** |  |  [optional] |
 |**facility** | [**Href**](Href.md) |  |  [optional] |
 |**id** | **UUID** |  |  [optional] |
-|**metro** | [**InterconnectionMetro**](InterconnectionMetro.md) |  |  [optional] |
+|**metro** | [**Metro**](Metro.md) |  |  [optional] |
```

Comment:
One of the v7.0.0 features that is enabled by default is a schema simplification that recognizes that `InterconnectionMetro` is really just a `Metro`